### PR TITLE
Explainer: make the idea clearer

### PR DESCRIPTION
**Pull request by bokand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,52 +2,61 @@
 
 ## Summary
 
-This is a proposal to enable users and authors to group multiple resources into a single link. An agent navigating to such a link will load and present all its resources in a grouping UI. We're calling such a link a "page collection":
+It's common for users browsing the web to open and refer to multiple pages in a single journey. For example, when doing research on a
+complex topic, comparing products when shopping, or planning a trip or outing. Browsers have adapted to this by introducing tabbed UIs, and
+later allowing grouping of related tabs. However, linking on the web remains 1-to-1; one link opens one page. This makes it difficult
```
Comment (hvanops) [inline README.md]: Restructuring suggestion (totally up to you to use or not) : You say browsers adapted for multi-page/-site journeys by introducing tabs and groups. Then maybe we can say "It's also common that during these complex, multi-page journeys, users want to include others by sharing the content and getting their thoughts to make the decision together. Yet- linking on the web, the only mechanism of sharing, remains 1-to-1; one link opens one page."

File: README.md [hunk 2]
```diff
@@ -2,52 +2,61 @@
...
-Page collections aims to allow the user agent to be more helpful in opening and presenting multiple related pages.
+A page collection allows the browser to be more helpful in managing and presenting groups of related pages.
 
-This explainer details ideas and challenges for this new form of linking. While some examples are provided, no grouping UI is suggested or mandated by this proposal; how such a link is presented is left to the discretion of the user agent.
+This explainer details ideas and challenges for this new form of linking. While some examples of "grouping UI" are provided, particularily
+with existing mechanisms such as tab groups, no specific UI is suggested or mandated in this proposal; how such a link is presented is left
+to the discretion of the user agent.
 
-### Summarized Web-Facing Changes
+### Non-Goals
```
Comment (hvanops) [inline README.md:30]: non-goals are great, and will help with the feedback we received on the v0 of this explainer. 
I'm wondering if we should also have a more explicit "goals" section too?

File: README.md (hunk 1 above)
Comment (bokand) [reply README.md]: Thanks, added this reworded slightly for brevity

File: README.md (hunk 2 above)
Comment (bokand) [reply README.md:30]: I think the summary is effectively the goals section so we'd be doing a lot of repetition and attention is limited so I'm trying (often unsuccessfully) to be as succinct as possible.

Comment (bokand) [reply README.md:30]: Though, if you think there is anything missing or understated, maybe we can emphasize it more in the summary?

**Pull request merged**